packages/portal-proto/src/components/InputEntityList/InputEntityList.tsx

```diff
@@ -269,7 +269,7 @@ const InputEntityList: React.FC<InputEntityListProps> = ({
           />
         )}
       </div>
-      <ButtonContainer>
+      <ButtonContainer data-testid="modal-button-container">
```

Comment:
Same.
If it's a button, this should start with `button-`

Reply:
As discussed it is not a button, but rather the bottom portion of the modal that contains buttons